tests/e2e/linalg_ext_ops/BUILD

```diff
@@ -63,6 +63,32 @@ iree_check_single_backend_test_suite(
     target_backend = "cuda",
 )
 
+iree_check_single_backend_test_suite(
+    name = "check_cuda-topk-split-reduction-double",
+    srcs = enforce_glob(
```

Comment:
nit: you probably don't need to use `enforce_glob` (it's only there to check that no file are missed but here you only care about enabling it for a top-k case)